examples/config-cve.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "0.1.0-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot"
+    },
+    "http": {
+        "address": "127.0.0.1",
+        "port": "8080"
+    },
+    "log": {
+        "level": "debug"
+    },
+    "extensions": {
+        "search": {
+            "cve": {
+                "periodicupdates": true
+            }
+        }
+    }
+}
```

Comment:
Make sure you put newlines at the end of all json, go files

Reply:
Ok